lib/css/components/_stacks-topbar.less

```diff
@@ -0,0 +1,366 @@
+@import (reference) "../exports/_stacks-exports.less";
+// TODO why is local less.js not finding these automatically?
+@import (reference) "../atomic/_stacks-typography.less";
+@import (reference) "../atomic/_stacks-misc.less";
+@import (reference) "../atomic/_stacks-width-height.less";
+@import (reference) "../atomic/_stacks-borders.less";
+
+// forced light topbar
+.s-topbar__light {
+    --theme-topbar-background-color:           @black-025;
+
+    // Search input
+    --theme-topbar-search-color:               @black-700;
+    --theme-topbar-search-background:          @white;
+    --theme-topbar-search-placeholder:         @black-200;
+    --theme-topbar-search-border:              @black-200;
+    --theme-topbar-search-border-focus:        @blue-300;
+    --theme-topbar-search-shadow-focus:        0 0 0 @su4 fade(@blue-500, 15%);
```

Comment:
This likely still needs to be aware of `var(--theme-primary)`. Borrow the approach we used on buttons? Will `0 0 0 @su4 var(--focus-ring);` cut it?

Reply:
Good call on this as well. I was using the value on Prod, but I like yours better.